tmv.h

```diff
@@ -48,6 +48,7 @@ extern int tmv_is_zero(tmv_t x);
 extern tmv_t tmv_sub(tmv_t a, tmv_t b);
 extern tmv_t tmv_zero(void);
 extern tmv_t correction_to_tmv(Integer64 c);
+extern Integer64 tmv_frac_to_correction(tmv_t x);
```

Comment:
It seems slightly ugly to have a tmv_frac_...() function, since every other function in the API handles a complete tmv value.  Is there any cleaner way we could do this?  (I can't immediately see a way that doesn't introduce unnecessary extra calculations.)